src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -1141,7 +1141,7 @@ public enum Platinum implements LogicCardInfo {
             energyCost G, C, C
             onAttack {
               damage 60
-              if(defending.hasPokePower || defending.hasPokeBody) {
```

Comment:
This one is complicated, but anything that affects PokéPower AND PokéBody should affect Pokémon Powers. (In Japanese, Pokémon Powers were actually errata'd to either a PokéBody or a PokéPower, so we may actually need to figure out what we want to do regarding that at some points since it's kind of important for Unlimited.)